tasties_app/models.py

```diff
@@ -46,6 +55,65 @@ class UnitChoices(models.TextChoices, Enum):
     )
     description = models.CharField(max_length=64, validators=[MinLengthValidator(1)])
 
+    def __str__(self):
+        return str(self.amount) + " " + self.measurement_unit + " " + self.description
+
+    @staticmethod
+    def is_enum_value(value):
```

Comment:
Yes. This class has the 'measurement_unit' attribute. This attribute can only get values from the enum 'UnitChoices'. In order to check if the value is in the enum you need to iterate over the enum values to find it. The method is being used as a validator (line 83).

Reply:
> Yes. This class has the 'measurement_unit' attribute. This attribute can only get values from the enum 'UnitChoices'. In order to check if the value is in the enum you need to iterate over the enum values to find it. The method is being used as a validator (line 83).

Please take a look at [this](https://stackoverflow.com/questions/43634618/how-do-i-test-if-int-value-exists-in-python-enum-without-using-try-catch), we might not have to use this method, or it could be made a bit simpler.